packages/config/src/node/loadConfig.ts

```diff
@@ -7,8 +7,8 @@ import { pathToFileURL } from "url";
 import os from "os";
 
 // In order of preference files are checked
-const configFiles = ["mud.config.js", "mud.config.mjs", "mud.config.ts", "mud.config.mts"];
-const TEMP_CONFIG = "mud.config.temp.mjs";
+const CONFIG_FILE = path.join(".mud", "expandedConfig.ts");
+const TEMP_CONFIG = path.join(".mud", "expandedConfig.temp.mjs");
```

Comment:
in this PR I expect `.mud` to just exist (which it would because templates have it, and it doesn't need to change)
I think we should have a `mudup` as an actual command rather than a script wrapper in templates, and it can then take care of `.mud` properly